source/openfisca-web-api/index.md

```diff
@@ -9,13 +9,22 @@
    config-openapi
 ```
 
-OpenFisca provides a web API package compatible with all country packages.
-Using a web interface, App Developers can access information and computations without installing anything locally.
+OpenFisca provides a web API package, compatible with all country packages.
+Using a web interface, app developers can access information and computations without installing anything locally.
 
 ## Public France API
 
-The latest version of the France web api is [`fr.openfisca.org/api/v21`](https://fr.openfisca.org/api/v21).
-Its endpoints are documented in [`fr.openfisca.org/legislation/swagger`](https://fr.openfisca.org/legislation/swagger).
+The latest version of the France web api is [`fr.openfisca.org/api/latest`](https://fr.openfisca.org/api/latest). Its endpoints are documented in [`fr.openfisca.org/legislation/swagger`](https://fr.openfisca.org/legislation/swagger). This API has no guarantee of stability over time. Building a web app on top of it is not recommended.
+
+The [DINSIC's OpenFisca Team](https://github.com/orgs/openfisca/teams/france-contrib-dinsic) maintains two stable versions of the Web API. Every 6 months, on January 1st and July 1st, the oldest stable API is shut down, and the current newest version of France is served instead. This means every Web API version is maintained for a year.
+
+Currently served Web API versions are listed in [https://fr.openfisca.org/api](https://fr.openfisca.org/api)
+
+These stable Web APIs are meant to be used for building minimum viable products, proofs of concept and prototypes. The idea is to help web app developers quickly test out new ideas, find their first users or secure funding.
```

Comment:
Why would one want to develop MVPs, POCs and protos using a *stable* API specifically, as opposed to the latest ? Conversely, what are we recommending people use `/latest` for, as opposed to stating (as we do) what *not* to use it for ?